DTM 1019.1.0 - Asset Location, Ordering of complex properties documentation (#1566)
SoftwareAG/c8y-docs: content/dtm/asset-types-bundle/asset-properties.md

<td style="text-align:left">Optional</td>
</tr>
<tr>
<td style="text-align:left"><b>Order</b></td>

> <td style="text-align:left">Used to display the complex properties in a specific order.</td>

Reviewer: ```suggestion
<td style="text-align:left">Used to define the order in which the keys of a complex property must be shown.</td>
```

![Complex asset property](/images/dtm/custom-property/dtm-property-library-complex-property.png)

{{< c8y-admon-info>}}

> If you have not mentioned any specific order in **Order** field, it is automatically prefilled with the order in which the key value pairs are added.

Reviewer 2: ```suggestion
If specific order is not mentioned in **Order** field then it is automatically prefilled with the order in which the key value pairs are added.
```

Reviewer: ```suggestion
If the **Order** field is left empty, the order is determined by the sequence in which the key value pairs are added and the field will be automatically filled on saving the asset property.
```

> If the **Order** field is left empty, the order is determined by the sequence in which the key value pairs are added and the field will be automatically filled on saving the asset property.

Reviewer 3: ```suggestion
If you do not fill out the **Order** field, the order is determined by the sequence in which the key value pairs are added and the field will be automatically filled on saving the asset property.
```

The **Location** property enables you to assign location (latitude and longitude) to an asset. You can set the default values of the latitude and longitude by entering the corresponding values or alternatively using the map view.

> To set the default values for the latitude and longitude using the map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. By default, the marker on the map is not displayed to users. To make the marker visible, users click on the map. Click or drag the marker to the preferred position to select the default value for latitude and longitude.

Reviewer: @Vagdevi1998 , please ensure the description below is as per the current implementation before using the suggestion as is.
```suggestion
To set the default values for the latitude and longitude using the map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. Initially, if the default latitude and longitude values are not present, the marker is not shown. To see the marker, click anywhere on the map. Click or drag the marker to the preferred position to select the default value for latitude and longitude.
```

Author: As discussed, I am keeping it same.

> To set the default values for the latitude and longitude using the map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. Initially, if the default latitude and longitude values are not present, the marker is not shown. To see the marker, click anywhere on the map. Click or drag the marker to the preferred position to select the default value for latitude and longitude.

Reviewer 3: ```suggestion
To set the default values for the latitude and longitude using the map view, click **Choose on Map** at the end of the section. Click the full screen icon at the top right corner of the map to view it in full screen. Without values for latitude and longitude, the marker is hidden. To see the marker, click anywhere on the map. Click or drag the marker to the preferred position to select the default value for latitude and longitude.
```

{{<c8y-admon-info>}}
The valid range for the latitude is -90 to +90 and valid range for the longitude is -180 to +180. <br>

> The marker on the map is not visible when either the latitude or longitude value is missing.<br>

Reviewer 2: There are 2 scenarios:
In view mode map itself will not be visible if lat or long is is not supplied
In edit mode map will be visible but marker wont be visible if lat or long is is not supplied
Please modify the statement accordingly

> The marker on the map is not visible when either the latitude or longitude value is missing.<br>

Reviewer: ```suggestion
The marker on the map is not shown when either the latitude or longitude value is missing.<br>
```

> The marker on the map is not shown when either the latitude or longitude value is missing.<br>

Reviewer 3: ```suggestion
The marker on the map is only visible when both values for latitude and longitude are provided.<br>
```
